protostar/testing/fuzzing/strategy_descriptor.py

```diff
@@ -1,11 +1,11 @@
 from abc import ABC, abstractmethod
-from typing import Callable
+from typing import Callable, Any
 
 from hypothesis.strategies import SearchStrategy
 from starkware.cairo.lang.compiler.ast.cairo_types import CairoType
 
-MappingFunction = Callable[[int], int]
-FilterFunction = Callable[[int], bool]
+MappingFunction = Callable[[Any], Any]
+FilterFunction = Callable[[Any], bool]
```

Comment:
Having `Any` here disables basically all checks for the args. Is that the only possible way?

Reply:
Eventually these types were doomed to use `Any` (they originally were when I wrote that, but during code review we changed this to `int`).

Technically we could use some generics here, @Radinyn you could experiment with that here.

Reply:
I mean, although the user doesn't see typing, it makes more sense in my opinion to use `Any`, after all the mapping function doesn't have to return the same type.